Sources/LogConsole/LogInterceptor.cs

```diff
@@ -10,6 +10,8 @@
 using System;
 using UnityEngine;
 
+using Logger = KSPDev.LogUtils.Logger;
```

Comment:
There is a bunch of mods that use it, and I didn't cleanup the code yet. Deprecating `KSPDev.LogUtils.Logger` is a nice to have feature. Just FYI.